.github/workflows/lint.yml

```diff
@@ -38,15 +38,14 @@ jobs:
         id: setup-python
       - name: Add problem matchers
         run: |
-          echo "::add-matcher::.github/workflows/matchers/pylint.json"
+          echo "::add-matcher::.github/workflows/matchers/clint.json"
           echo "::add-matcher::.github/workflows/matchers/format.json"
           echo "::add-matcher::.github/workflows/matchers/ruff.json"
       - uses: ./.github/actions/cache-pip
       - name: Install dependencies
         run: |
           python -m venv .venv
           source .venv/bin/activate
-          source ./dev/install-common-deps.sh --ml
```

Comment:
Pylint also checks class inheritance, so we needed to install ML packages mlflow depends on.